pkg/reconciler/adapter.go

```diff
@@ -309,3 +322,40 @@ func MaybeAppendValueFromEnvVar(envs []corev1.EnvVar, key string, valueFrom v1al
 
 	return envs
 }
+
+// adapterOverrideOptions applies adapter override parameters depending on
+// deployment type.
+func adapterOverrideOptions(overrides *v1alpha1.AdapterOverrides) []resource.ObjectOption {
+	var opts []resource.ObjectOption
+
+	opts = append(opts, func(object interface{}) {
+		if _, ok := object.(*servingv1.Service); ok {
+			if public := overrides.Public; public != nil && *public {
+				resource.VisibilityPublic(object)
+			} else {
+				resource.VisibilityClusterLocal(object)
+			}
+		}
+	})
+
+	if overrides.Resources != nil {
+		opts = append(opts, resource.Requests(toQuantity(overrides.Resources.Requests)))
+		opts = append(opts, resource.Limits(toQuantity(overrides.Resources.Limits)))
+	}
+
+	return opts
+}
+
+func toQuantity(resources corev1.ResourceList) (kres.Quantity, kres.Quantity) {
+	var cpu, memory kres.Quantity
+
+	for k, v := range resources {
+		switch k {
+		case "cpu":
```

Comment:
nit: Kubernetes has a [constant](https://github.com/triggermesh/triggermesh/blob/7547c4b582e34706014a3aaa5f99d4e48e8050da/pkg/reconciler/resource/container.go#L228-L235) defined for that: `corev1.ResourceCPU`